HomeWork/src/main/java/HomeWork5/HomeWork5.java

```diff
@@ -24,8 +23,8 @@ public static void arrayWords () {
         Scanner scanner = new Scanner(System.in);
```

Comment:
The name `arrayWords()` does not give a clear picture of what it is doing. Would be better to rename to something like `startWordGuessingGame`